mne_bids/read.py

```diff
@@ -112,58 +211,30 @@ def read_raw_bids(bids_fname, bids_root, verbose=True):
 
     config = None
     if ext in ('.fif', '.ds', '.vhdr', '.edf', '.bdf', '.set', '.sqd', '.con'):
-        bids_fname = op.join(kind_dir,
-                             bids_basename + '_%s%s' % (kind, ext))
+        bids_fpath = op.join(kind_dir,
+                             bids_basename + '_{}{}'.format(kind, ext))
```

Comment:
please avoid cosmetic changes in a PR. It slows down the review for little gain.